test/test_fx.py

```diff
@@ -1621,6 +1621,46 @@ def forward(self):
         m = FooBar1234()
         self.checkGraphModule(m, ())
 
+    def test_torchbind_class_attribute_in_fx_tensor_arg(self):
+        if TEST_WITH_ROCM or IS_SANDCASTLE or IS_WINDOWS or IS_MACOS:
+            self.skipTest("torch.classes._TorchScriptTesting._ReLUClass is registered, skipping")
```

Comment:
This will probably time out in fbcode (see: D26838029). I suggest using `@unittest.skipIf` instead.

Reply:
Actually I think the problem I'm referring to only happens in `test_jit.py`, you can probably ignore this.